src/main/java/com/google/sps/data/Challenge.java

```diff
@@ -0,0 +1,37 @@
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Challenge {
+  private String challenge_type;
+  private List <String> steps;
+
+  public Challenge(String type, String name, List steps){  
```

Comment:
"name" is passed as a parameter but never used

Reply:
Thank you. I missed that.